compiler/rustc_codegen_llvm/src/asm.rs

```diff
@@ -322,6 +322,7 @@ impl AsmBuilderMethods<'tcx> for Builder<'a, 'll, 'tcx> {
                 InlineAsmArch::SpirV => {}
                 InlineAsmArch::Wasm32 | InlineAsmArch::Wasm64 => {}
                 InlineAsmArch::Bpf => {}
+                InlineAsmArch::Avr => {}
```

Comment:
I had to look this up in LLVM's internals, but the clobber for the status register is `"~{sreg}"`, which should be marked as clobbered when `preserves_flags` is not set. Also the documentation in the unstable book should be updated to reflect this.

Reply:
I've added that.

So many of the other platforms have an empty branch here it was easy to not spend much thought on this clause. But the realization that I missed this made be worried there might be more.  I just went through again and checked for other copy-pasta, didn't notice any concerns.

The one other place we have a null implementation that could potentially have code is `suggest_modifier` -- that one I did consider if there was a heuristic to use to know which byte you mean.  But in the end it seems equally likely you'd want either byte, and there doesn't seem to be a way to suggest *two* modifiers.